client-v2/src/actions/ArticleFragments.ts

```diff
@@ -54,6 +57,11 @@ type InsertThunkActionCreator = (
   removeAction: Action | null
 ) => ThunkResult<void>;
 
+const getEditingMode = (state: State): ClipboardType => {
```

Comment:
Can we use the `select` prefix here to make it clear this is operating on the app state?